Backend/dev/migrations/mock-registry/1-adding-subscriber-metadata.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE atlas_registry.subscriber ADD COLUMN image_url text;
+ALTER TABLE atlas_registry.subscriber ADD COLUMN "name" text;
```

Comment:
why is name in quotes?

Reply:
Name is a reserved keyword in SQL. So will have to enclose it in quotes.